integration_test/testutil_test.go

```diff
@@ -283,6 +285,14 @@ func (it *ITest) initializeIndex() {
 	if err := cmd.Run(); err != nil {
 		it.t.Fatalf("cannot restore index from cache: %s", err)
 	}
+
+	for _, e := range it.env {
+		if strings.Contains(e, constants.EnableMultiIndexSwitch) {
```

Comment:
Can you leave a todo comment as a reminder to simplify this setup when krew has fully migrated to the new layout?